src/Articles.jsx

```diff
@@ -69,7 +69,7 @@ export default function Articles() {
 				Order:{" "}
 				<select
 					id="order-by-select"
-					onChange={(e) => setSortOrder(e.target.value)}
+					onChange={(e) => SetSortOrder(e.target.value)}
```

Comment:
This looks like it is working however as your article cards don't currently display any date, vote or comment count, it is difficult for the user to tell if the articles have actually been sorted by what they have selected - they can only tell that something has happened as the page has refreshed and there may be a different article at the top. I'd recommend adding some more info to your article cards to show this clearly